5.6/apache/docker-yii2-php.conf

```diff
@@ -0,0 +1,27 @@
+<FilesMatch \.php$>
+        SetHandler application/x-httpd-php
+</FilesMatch>
+
+DirectoryIndex disabled
+DirectoryIndex index.php index.html
```

Comment:
I was wondering the same when I copied this from the docker/php Dockerfile. It's used to reset any previous calls to `DirectoryIndex`. From the [Apache manual](https://httpd.apache.org/docs/current/mod/mod_dir.html#directoryindex):

> Note: Multiple `DirectoryIndex` directives within the same context will add to the list of resources to look for rather than replace

and from the example there:

```
# Example C: To replace the list, you must explicitly reset it first:
# In this example, only index.php will remain as an index resource.
<Directory "/foo">
    DirectoryIndex index.html
    DirectoryIndex disabled
    DirectoryIndex index.php
</Directory>
```